.github/workflows/release.yml

```diff
@@ -0,0 +1,78 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+name: Release
+on:
+  push:
+    tags:
+      - kani-*
+
+jobs:
+  Release:
+    name: Release
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Get version
+        run: |
+          # pkgid is something like file:///home/ubuntu/rmc#kani-verifier:0.1.0
```

Comment:
Should this be `file:///home/ubuntu/kani#kani-verifier:0.1.0` or is there some GitHub Action limitation that forces the original name?

Reply:
Just giving an example of what the output of the command is, in CI it'll be different yeah.